### ISSUE TITLE
Dashboard  Open/Close all

### ISSUE THREAD

**Issue by renato-mmex** [moneymanagerex/moneymanagerex#6438]
Recently, some users have had problems with the display of accounts in the dashboard.
I would suggest that you add an Open/Close all function.
.
<img width="184" alt="image" src="https://github.com/moneymanagerex/moneymanagerex/assets/76920138/1a43b87b-5b54-4977-b4ea-c426fa2d4e47">

**Comment by n-stein** [comment]
> I wonder if a right click menu that had "Expand all/Collapse all" would help? Currently right click just opens Theme Manager,

I was going to suggest the same. The right click on 'Dashboard' node opening the theme manager seems unnecessary (and not very intuitive), so a simple right click menu as you said would be an improvement.

**Comment by renato-mmex** [comment]
A right-click is not self-explanatory.
In addition, the arrows are not very legible on some screens (especially on high-resolution monitors, e.g. 4k), if these arrows were bold, the problem would probably be smaller.

**Comment by whalley** [comment]
> In addition, the arrows are not very legible on some screens (especially on high-resolution monitors, e.g. 4k), if these arrows were bold, the problem would probably be smaller.

Agreed, but it seems we have little control over this as the wxWidgets library is in full control of this.

**Comment by rmelillo76** [comment]
Hey @renato-mmex, were there forum posts we can reference for more details? I've never had this problem, but I vaguely remember an old issue where the tree could be accidentally collapsed, showing the very top-level Dashboard node and it caused issues for some users, but that was resolved. I may be fuzzy on some of those details though.
I wonder if a right click menu that had "Expand all/Collapse all" would help? Currently right click just opens Theme Manager, 


> Perhaps alongside the right-click menu we also add to the toolbar 'View' menu to allow all items in the Navigation tree to opened or collappsed.

I came back here after I saw the other replies via email to add that very suggestion!
